graphql-codegen.yml

```diff
@@ -1,11 +1,10 @@
-
-
 overwrite: true
-schema: "https://api.thegraph.com/subgraphs/name/dxgraphs/swapr-xdai-v2"
+schema: 'https://api.thegraph.com/subgraphs/name/dxgraphs/swapr-xdai-v2'
 documents: 'src/**/!(*.d).{ts,tsx}'
 generates:
   ./src/generated/graphql/index.ts:
     plugins:
       - typescript
       - typescript-operations
-      - typescript-graphql-request
\ No newline at end of file
+      - typescript-graphql-request
+hooks: { afterOneFileWrite: ['sed -i -e"s|graphql-request/dist/types\.dom|graphql-request/src/types.dom|g"'] }
```

Comment:
Why are we adding this?

Reply:
Because `graphql-request` generated code in the version we use, is trying to import the DOM from the wrong path

https://github.com/dotansimha/graphql-code-generator-community/issues/501#issuecomment-1817878158

It's an issue persisting up to the next major version (v6.1.0) than the one we're using (v5.1.0)